cerebrium/environments/config-files.mdx

```diff
@@ -0,0 +1,155 @@
+
+The hardware parameters section is where you can define the specifications of the machine you would like to use for your deployment. This allows you to tailor your deployment to your specific needs, optimising for cost or performance as you see fit.  
+These parameters are specified under the `cerebrium.hardware` section of your config file.
+
+The available hardware parameters in your config are:
+| parameter | description | type | default |
+| --- | --- | --- | --- |
+| `gpu` | The GPU you would like to use. | string | AMPERE_A5000 |
+| `cpu` | The number of CPU cores to use. | int | 2 |
+| `memory` | The amount of Memory to use in GB. | float | 14.5 |
+| `gpu_count` | The number of GPUs to specify. | int | 1 |
+
+## Scaling Parameters
+
+This section lets you configure how you would like your deployment to scale.  
+You can use these parameters to control the minimum and maximum number of replicas to run, as well as the cooldown period between requests. For example, you could increase your cooldown time or even set a minimum number of replicas to run, increasing availability and avoiding cold starts.  
+Or, if you are looking to reduce your costs, you could decrease your cooldown time or set a maximum number of replicas to run, reducing the amount of time your deployment is running.  
+These parameters are specified under the `cerebrium.scaling` section of your config file.
+
+The available scaling parameters in your config are:
+| parameter | description | type | default |
+| --- | --- | --- | --- |
+| `min_replicas` | The minimum number of replicas to run at all times. | int | 0 |
+| `max_replicas` | The maximum number of replicas to scale to. | int | plan limit |
+| `cooldown` | The number of seconds to keep your model warm after each request. It resets after every request ends. | int | 60 |
+
+## Adding Dependencies
+
+- What dependencies can I add?
+- How do I add them?
+- How are dependencies handled in Cerebrium?
+- requirements.txt into toml. What is the priority?
+
+The dependencies section of your config file is where you can specify any dependencies you would like to install in your deployment. We support pip, conda and apt dependencies and you can specify each of these in their relevant subsection of the dependencies section.
+
+For each dependency type, you can specify the name of the package you would like to install and the version constraints. If you do not want to specify any version constraints, you can use the `latest` keyword to install the latest version of the package.
+
+If you have an existing **requirements.txt**, **pkglist.txt** or **conda_pkglist.txt**, we'll prompt you to automatically integrate these into your config file when you run `cerebrium deploy`.
+
+### pip
+
+Your pip dependencies are specified under the `cerebrium.dependencies.pip` section of your config file.
+An example of a pip dependency is shown below:
+
+```toml
+[cerebrium.dependencies.pip]
+torch = ">=2.0.0"
+numpy = "latest"
```

Comment:
Uhm I wanted to chat about this, why did you do `torch = ">= 2.0.0"` instead of `torch >= "2.0.0"? Seems like a very weird decision

Reply:
Yeah, it's not ideal. 
but it's the way it has to be done because of the toml language. 
Look at the way versions are defined in poetry. 

So the toml reads in as a dictionary with key, val pairs. The version is the value.